verb/improved.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": {
+    "en-US": "Improve"
```

Comment:
@bbeversdorf Suggest this be labeled as "Improved"

Reply:
They should probably both be present tense or past tense.
https://registry.tincanapi.com/#home/verbs seems mixed now.
Worsen and Improve
or
Worsened and Improved